spec/models/like_spec.rb

```diff
@@ -0,0 +1,15 @@
+require './spec/rails_helper'
+
+RSpec.describe Like, type: :model do
+  subject { Like.create(AuthorId: 1, PostId: 2) }
+
```

Comment:
- [ ] It would be better if we test all our custom methods. For example; `update_the_likes_counter` method is not tested. Let us also test the method in our `comment` model

Reply:
Ok, I am including that also.

Reply:
Though it has been done in post model.

Reply:
well resolved! ✔️